### DISCUSSION TITLE
Feature vs. Addon

### DISCUSSION THREAD

**Discussion by DogCatPuppyLover** [ScratchAddons/ScratchAddons#2152] [Help] [answered]
What is the difference between these two categories? I was going to suggest an improvement to an addon; would that go into Feature or New Addon? Thanks!

✨ Feature

➕ New addon

**Comment by TheColaber** [comment, marked as answer]
✨ Feature could be a new feature to an addon, settings page, etc.
➕ New addon is a new addon.
